flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -606,6 +614,61 @@ class FlxGamepad implements IFlxDestroyable
 		return getAnalogYAxisValue(Stick);
 	}
 
+	@:allow(flixel.input.gamepad.FlxGamepadManager)
+	private function handleAxisMove(axis:Int, newVal:Float, oldVal:Float)
```

Comment:
Abbreviating `value` to `val` seems a bit silly, barely saves any chars and definitely doesn't help readability.